scripts/ci-cluster-provision.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a wrapper for cross-cloud's provision.sh
+# targeted to use in Circle CI environment
+
+export TF_VAR_packet_project_id="${PACKET_PROJECT_ID}"
+pushd /cncf
+./provision.sh "$1" nsm-ci-"${CIRCLE_WORKFLOW_ID:0:8}" file
```

Comment:
Does this imply that we have a maximum of 8 concurrent clusters to test against?